src/omv/sensors/genx320.c

```diff
@@ -231,7 +235,18 @@ static int set_framerate(omv_csi_t *csi, int framerate) {
         return -1;
     }
 
-    psee_sensor_write(EHC_INTEGRATION_PERIOD, EVT_SCALE_PERIOD(us));
+    psee_sensor_write(EHC_INTEGRATION_PERIOD, us);
```

Comment:
The new ISSD internally makes the camera timebase not need scaling by the clock anymore.